Index: partclone-0.2.38/config.h.in
===================================================================
--- partclone-0.2.38.orig/config.h.in
+++ partclone-0.2.38/config.h.in
@@ -76,7 +76,7 @@
 /* Define to 1 if you have the <unistd.h> header file. */
 #undef HAVE_UNISTD_H
 
-/* Define to 1 if you have the <xfs/libxfs.h> header file. */
+/* Define to 1 if you have the <xfs/xfs.h> header file. */
 #undef HAVE_XFS_LIBXFS_H
 
 /* "define ntfs version" */
Index: partclone-0.2.38/configure.ac
===================================================================
--- partclone-0.2.38.orig/configure.ac
+++ partclone-0.2.38/configure.ac
@@ -111,7 +111,7 @@ AC_CHECK_LIB([rt], [aio_init], true,
 
 dnl Check for XFS 
 AC_CHECKING([ for XFS Library and Header files ... ])
-AC_CHECK_HEADERS([xfs/libxfs.h], ,
+AC_CHECK_HEADERS([xfs/xfs.h], ,
    AC_MSG_ERROR([*** XFS header (libxfs.h) not found])
 )
 supported_fs=$supported_fs" xfs"
Index: partclone-0.2.38/src/xfsclone.c
===================================================================
--- partclone-0.2.38.orig/src/xfsclone.c
+++ partclone-0.2.38/src/xfsclone.c
@@ -12,7 +12,7 @@
  */
 
 #include <sys/param.h>
-#include <xfs/libxfs.h>
+#include <xfs/xfs.h>
 #include <sys/stat.h>
 #include <stdio.h>
 #include <fcntl.h>
Index: partclone-0.2.38/src/xfsclone.h
===================================================================
--- partclone-0.2.38.orig/src/xfsclone.h
+++ partclone-0.2.38/src/xfsclone.h
@@ -11,7 +11,7 @@
  * (at your option) any later version.
  */
 
-#include <xfs/libxfs.h>
+#include <xfs/xfs.h>
 
 /*
  * An on-disk allocation group header is composed of 4 structures,
